sources/Database/ArkadiaDbContext.cs

```diff
@@ -24,6 +24,22 @@ public ArkadiaDbContext(DbContextOptions<ArkadiaDbContext> options) : base(optio
 		public DbSet<Guild> Guilds { get; set; }
 		public DbSet<YoutubeSubscription> YoutubeSubscriptions { get; set; }
 
+		protected override void OnModelCreating(ModelBuilder modelBuilder)
+		{
+			modelBuilder.Entity<CdnEntry>()
+				.Property(entry => entry.LastModifiedAt)
+				.HasConversion(
+					date => date.Ticks,
+					ticks => new DateTime(ticks)
+				);
+			modelBuilder.Entity<YoutubeSubscription>()
+				.Property(entry => entry.ExpiresAt)
+				.HasConversion(
+					date => date.Ticks,
+					ticks => new DateTime(ticks)
+				);
```

Comment:
Do we need those? I don't think I have them in Lynx, and it works just fine.

Reply:
apparently we do:
> Exception data:
    Severity: ERROR
    SqlState: 42804
    MessageText: column "expires_at" cannot be cast automatically to type timestamp without time zone
    Hint: You might need to specify "USING expires_at::timestamp without time zone".
    File: tablecmds.c
    Line: 10741
    Routine: ATPrepAlterColumnType
42804: column "expires_at" cannot be cast automatically to type timestamp without time zone

Reply:
Actually, we don't. It's complaining because by default, DateTime generates a `TIMESTAMP WITHOUT TIME ZONE` instead of a `TIMESTAMP WITH TIME ZONE`. You can fix this error by using UTC.